fuse/eval/metrics/sequence_gen/metrics_seq_gen_common.py

```diff
@@ -98,19 +104,29 @@ def _count_seq_and_tokens_update(
     # to save GPU memory
     encoder_input = encoder_input.detach()
 
-    if ignore_index is not None:
-        mask = encoder_input.ne(ignore_index)
-    else:
-        mask = torch.ones_like(encoder_input, dtype=torch.bool)
+    mask = get_mask(encoder_input, ignore_index)
 
     seq_num = torch.tensor(
         mask.shape[0], dtype=torch.int64, device=encoder_input.device
     )
     token_num = mask.sum().to(dtype=torch.int64)
 
+    if decoder_input_key is not None and decoder_input_key in batch_dict:
+        decoder_input = batch_dict[decoder_input_key].detach()
+        mask2 = get_mask(decoder_input, ignore_index)
+        assert mask2.shape[0] == mask.shape[0]
+        token_num += mask2.sum().to(dtype=torch.int64)
```

Comment:
Just making sure:
Here's the support for counting the labels tokens as well?

If so, best to double check with @mosheraboh :)

Reply:
@SagiPolaczek I updated @mosheraboh, but better confirm the change with him.